packages/bpk-component-datatable/src/bpk-data-table.scss

```diff
@@ -0,0 +1,69 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins';
+
+.bpk-data-table {
+  width: 100%;
+
+  &__headerColumn {
+    display: flex;
+    height: 100%;
+    flex-direction: row;
+    align-items: center;
+
+    &:first-child {
+      padding: 0 $bpk-spacing-base;
+    }
+
+    &:last-child {
+      padding: 0 $bpk-spacing-base;
+    }
+
+    &:hover {
+      cursor: pointer;
+    }
+  }
+
+  &__row {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    border: $bpk-border-size-sm solid $bpk-color-gray-50;
+    border-bottom: none;
+    color: $bpk-color-gray-700;
+    font-weight: normal;
+
+    &:last-child {
+      border-bottom: $bpk-border-size-sm solid $bpk-color-gray-50;
+    }
+
+    &:not(.bpk-data-table__headerRow):hover {
```

Comment:
This should be `.bpk-data-table__header-row.

Just noticed... there are no styles associated with this class - do you mean `.bpk-data-table__row` (defined on [line 43](https://github.com/Skyscanner/backpack/pull/370/files#diff-55ba15298ffc9780198b9a01011f4642R43))

Reply:
yeah - no styles associated, but the class is programatically set on header rows to give it access to this rule